google-cloud-bigtable/lib/google/cloud/bigtable/admin.rb

```diff
@@ -22,7 +22,7 @@ module Bigtable
       # rubocop:disable LineLength
 
       ##
-      # # Ruby Client for Cloud Bigtable Admin API ([GA](https://github.com/googleapis/google-cloud-ruby#versioning))
+      # # Ruby Client for Cloud Bigtable Admin API ([Beta](https://github.com/googleapis/google-cloud-ruby#versioning))
```

Comment:
@dazuma I guess I need to update synth.py for the change to `GA`?

Reply:
I'm getting the upstream setting fixed, so we shouldn't need to synth-hack this.

Reply:
Got it, thanks for doing that!